web/packages/teleport/src/services/joinToken/joinToken.ts

```diff
@@ -98,12 +99,20 @@ class JoinTokenService {
       .then(makeJoinToken);
   }
 
-  async createJoinToken(req: CreateJoinTokenRequest) {
-    return api.post(cfg.getJoinTokensUrl(), req).then(makeJoinToken);
+  async createJoinToken(
+    req: CreateJoinTokenRequest,
+    mfaResponse: MfaChallengeResponse
```

Comment:
so funny enough this mfaResponse is required @ravicious, since the multi endpoint call is in the proxy this time 😅